lib/src/main/java/com/google/android/material/composethemeadapter/TypedArrayUtils.kt

```diff
@@ -157,18 +157,18 @@ internal fun parseShapeAppearance(
         when (a.getInt(R.styleable.ComposeThemeAdapterShapeAppearance_cornerFamily, 0)) {
             0 -> {
                 RoundedCornerShape(
-                    topLeft = cornerSizeTL ?: defaultCornerSize ?: fallbackShape.topLeft,
-                    topRight = cornerSizeTR ?: defaultCornerSize ?: fallbackShape.topRight,
-                    bottomRight = cornerSizeBR ?: defaultCornerSize ?: fallbackShape.bottomRight,
-                    bottomLeft = cornerSizeBL ?: defaultCornerSize ?: fallbackShape.bottomLeft
+                    topStart = cornerSizeTL ?: defaultCornerSize ?: fallbackShape.topStart,
```

Comment:
Probably need to convert left/right to start/end here. Fine for now, but should be fixed before alpha12

Reply:
Done